homeworks/sviatoslav.taranenko_SviatoslavTaranenko/GamesProject/style.css

```diff
@@ -0,0 +1,29 @@
+.about__all {
+  height: 479px;
+  weigth: 1175px;
+}
+
+.about__text {
+  height: 479px;
+  width: 575px;
```

Comment:
If the size of content could be changed (e.g. text, not an image), would be better not to fix both `width` and `height`. It could lead to layout beaks.
<img width="1644" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/e8c3fcb1-3a96-470b-b5f2-e1249e401da8">